.github/workflows/pull-request.yml

```
@@ -560,4 +560,27 @@ jobs:
       - name: Run docstring compliance check
         run: |
           source venv/bin/activate
-          python .github/workflows/scripts/check_docstrings.py --directories .github
+          python .github/workflows/scripts/check_docstrings.py --directories .github   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)